airflow/providers/apache/druid/README.md

```diff
@@ -0,0 +1,125 @@
+ under the License.
+ -->
+
+
+# Package apache-airflow-backport-providers-apache-druid
+
+Release: 2020.05.19
+
+**Table of contents**
+
+- [Backport package](#backport-package)
+- [Installation](#installation)
+- [Compatibility](#compatibility)
+- [PIP requirements](#pip-requirements)
+- [Cross provider package dependencies](#cross-provider-package-dependencies)
+- [Provider class summary](#provider-class-summary)
+    - [Operators](#operators)
+        - [Moved operators](#moved-operators)
+    - [Hooks](#hooks)
+        - [Moved hooks](#moved-hooks)
+- [Releases](#releases)
+    - [Release 2020.05.19](#release-20200519)
+
+## Backport package
+
+This is a backport providers package for `apache.druid` provider. All classes for this provider package
+are in `airflow.providers.apache.druid` python package.
+
+## Installation
+
+You can install this package on top of an existing airflow 1.10.* installation via
+`pip install apache-airflow-backport-providers-apache-druid`
+
+## Compatibility
+
+For full compatibility and test status of the backport packages check
+[Airflow Backport Package Compatibility](https://cwiki.apache.org/confluence/display/AIRFLOW/Backported+providers+packages+for+Airflow+1.10.*+series)
+
+## PIP requirements
+
+| PIP package   | Version required   |
+|:--------------|:-------------------|
+| pydruid       | &gt;=0.4.1,&lt;=0.5.8    |
+
+## Cross provider package dependencies
+
+Those are dependencies that might be needed in order to use all the features of the package.
+You need to install the specified backport providers package in order to use them.
+
+You can install such cross-provider dependencies when installing from PyPI. For example:
+
+```bash
+pip install apache-airflow-backport-providers-apache-druid[apache.hive]
+```
+
+| Dependent package                                                                                                            | Extra       |
+|:-----------------------------------------------------------------------------------------------------------------------------|:------------|
+| [apache-airflow-backport-providers-apache-hive](https://github.com/apache/airflow/tree/master/airflow/providers/apache/hive) | apache.hive |
+
+# Provider class summary
+
+All classes in Airflow 2.0 are in `airflow.providers.apache.druid` package.
+
+
+## Operators
+
+
+
+
+### Moved operators
+
+| Airflow 2.0 operators: `airflow.providers.apache.druid` package                                                                                        | Airflow 1.10.* previous location (usually `airflow.contrib`)                                                                                               |
+|:-------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
The intent with these README files is that they will become the README of the uploaded dist on PyPi, correct?

Have we tested if PyPi's markdown renderer supports these type of tables?

Reply:
Good point. Will check

Reply:
Yep: https://test.pypi.org/project/apache-airflow-backport-providers-google/2020.5.19rc1/

Reply:
Just the TOC does not work, but I think this is ok.

Reply:
It still works on GitHub

Reply:
Is it difficult to strip them out of the version we bundle in the dist?

Reply:
Not at all: https://test.pypi.org/project/apache-airflow-backport-providers-google/2020.5.19rc4/